tests/test_iam.py

```diff
@@ -522,4 +523,21 @@ def test_sqs_policies(self):
                 policies, [False, True, True, False,
                            False, False, False, False]):
             violations = check_cross_account(p, set(['221800032964']))
-            self.assertEqual(bool(violations), expected)
\ No newline at end of file
+            self.assertEqual(bool(violations), expected)
+
+
+class IamUserKeyLastUsedtest(BaseTest):
+
+    def test_last_used_user_key(self):
+        self.patch(
+            UserAccessKeyLastUsed, 'executor_factory', MainThreadExecutor)
+        session = self.replay_flight_data('test_iam_last_used_user_key')
+        policy = self.load_policy({
+            'name': 'iam-user-key-last-used',
+            'resource': 'iam-user',
+            'filters': [{
+                'type': 'key-last-used',
+                'days': 14}]
+        }, session_factory=session)
+        resources = policy.run()
+        self.assertEqual(len(resources), 3)
```

Comment:
I'm using a seperate AWS account to generate the test results in regards to this one. Will add the additional test assertions. Should have thought of that.